doc/manuals/admin/cli.md

```diff
@@ -87,6 +87,12 @@ The list of available options is the following:
     contextBroker log file, instead of re-creating it when the
     broker starts.
 -   **--silent**. Suppress all log output except errors.
```

Comment:
--silent should be deprecated, thus removed from the documentation and (if possible) hiden with -u and -U. A comment about this should be included in deprecated.md.

Until definitevely removed, --silent should be supported as equivlanet of `-logLevel ERROR`.